blocks/library/quote/index.js

```diff
@@ -20,6 +20,9 @@ registerBlock( 'core/quote', {
 		citation: html( 'footer' ),
 		style: node => {
 			const value = attr( 'blockquote', 'class' )( node );
+			if ( ! value ) {
+				return 1;
```

Comment:
With this, there's three ways we're enforcing a default value for `style`.

- Explicitly return `1` from `style` parsing
- Return `null` from style parsing and fallback using `|| 1` in `edit` and `save`
- Destructure with `style = 1` default in `controls.isActive`

I think we ought to consolidate on one. My preference would be toward returning `undefined` in the `style` parsing and using destructuring with default value.